pyspark/bigdl/contrib/onnx2bigdl/load_graph.py

```diff
@@ -0,0 +1,131 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import numpy as np
+from bigdl.nn.onnx.layer import *
+from bigdl.nn.layer import Identity, Model
+from .ops_mapping import _convert_map as convert_map
+
+
+class OnnxGraph(object):
```

Comment:
it seems to me that this class is not a modeling for `ONNX` , still more like a utility and the class member defined below are just intermediate state for loading model only. We don't even need to model onnx as it's already provide by onnx (below  `onnx.load_model(file_path)`).  Can we combine the two classes together as one like `ONNXLoader` ?